go/test/endtoend/mysqlctld/mysqlctld_test.go

```diff
@@ -135,30 +134,11 @@ func initCluster(shardNames []string, totalTabletsRequired int) error {
 	return nil
 }
 
-const defaultOperationTimeout = 300 * time.Second
-const defaultRetryDelay = 3 * time.Second
-
-func waitForMysqlctldShutdown(t *testing.T, tab *cluster.Vttablet) bool {
-	tmr := time.NewTimer(defaultOperationTimeout)
-	defer tmr.Stop()
-	for {
-		if tab.MysqlctldProcess.HasShutdown() {
-			return true
-		}
-		select {
-		case <-tmr.C:
-			return false
-		default:
-		}
-		time.Sleep(defaultRetryDelay)
-	}
-}
-
 func TestRestart(t *testing.T) {
 	defer cluster.PanicHandler(t)
 	err := primaryTablet.MysqlctldProcess.Stop()
 	require.Nil(t, err)
-	require.Truef(t, waitForMysqlctldShutdown(t, primaryTablet), "Mysqlctld has not stopped after %s", defaultOperationTimeout)
+	require.Truef(t, primaryTablet.MysqlctldProcess.WaitForMysqlCtldShutdown(), "Mysqlctld has not stopped...")
```

Comment:
nit: why did we remove the part of the message that tells us how long we waited?

Reply:
because I moved it under cluster package and over there there was already a declared 'defaultOperationTimeout' variable which is used internally (not exported). I didn't feel like exporting it just for the sake of logging.